trust-store-trust-policy-specification.md

```diff
@@ -221,16 +216,16 @@ The RDN consists of an attribute type name followed by an equal sign and the str
 
 - Trust anchor MUST support a full and partial list of all the attribute types present in [subject DN](https://www.rfc-editor.org/rfc/rfc5280.html#section-4.1.2.6) of x509 certificate.
 - If the subject DN of the signing certificate is used in the trust anchor, then it MUST meet the following requirements:
-  - The value of `trustAnchors` MUST begin with `subject:` followed by comma-separated one or more RDNs.
-    For example, `subject: C=${country}, ST=${state}, L=${locallity}, O={organization}, OU=${organization-unit}, CN=${common-name}`.
+  - The value of `trustedIdentities` MUST begin with `x509.subject:` followed by comma-separated one or more RDNs.
+    For example, `x509.subject: C=${country}, ST=${state}, L=${locallity}, O={organization}, OU=${organization-unit}, CN=${common-name}`.
   - Trust anchor MUST contain country (CN), state Or province (ST), and organization (O) RDNs.
     All other RDNs are optional.
-    The minimal possible trust anchor is `subject: C=${country}, ST=${state}, O={organization}`,
+    The minimal possible trust anchor is `x509.subject: C=${country}, ST=${state}, O={organization}`,
   - Trust anchor MUST NOT have overlapping values.
```

Comment:
What is the expected error behavior when this happens.  Does the first one win or does functionality stop, or does the attempt to update this policy fail?

Reply:
Good point, I'll clarify that signature verification fails with "invalid policy".